common/app/common/NewNavigation.scala

```diff
@@ -422,7 +422,7 @@ object NewNavigation {
       }
 
       if (sectionList.isEmpty) {
-        ("", List())
+        ("News", News.getEditionalisedNavLinks(edition).drop(1))
```

Comment:
The first item in the list for each section is the same link as used in either News, Opinion, Sport, Arts, Life here:
![image](https://cloud.githubusercontent.com/assets/8774970/20665782/da18dcee-b558-11e6-9369-8fb06cf4edec.png)

No sense having the links twice! I may need a comment explaining this though. Thanks for asking